tasks/jasmine.js

```diff
@@ -194,7 +194,7 @@ module.exports = function(grunt) {
 
     phantomjs.on('console', function(msg) {
       thisRun.cleanConsole = false;
-      if (options.display === 'full') {
+      if (options.display === 'full' || options.showLogging) {
```

Comment:
I guess my main concern is that this requires people to add `showLogging: true` in order to see log messages in "full" mode. My approach preserved existing behavior, allowing the desired improvement through a new option.